src/main/java/seedu/address/model/student/StudentMatchesQueryPredicate.java

```diff
@@ -20,12 +20,12 @@ public StudentMatchesQueryPredicate(String keyword) {
 
     @Override
     public boolean test(Student student) {
-        return FuzzySearch.partialRatio(student.getStudentNumber().toString().toLowerCase(), keyword) > MATCH_RATIO
-            || FuzzySearch.partialRatio(student.getName().toString().toLowerCase(), keyword) > MATCH_RATIO
-            || FuzzySearch.partialRatio(student.getEmail().toString().toLowerCase(), keyword) > MATCH_RATIO
-            || FuzzySearch.partialRatio(student.getGroupName().toString().toLowerCase(), keyword) > MATCH_RATIO
+        return FuzzySearch.ratio(student.getStudentNumber().toString().toLowerCase(), keyword) > MATCH_RATIO
```

Comment:
Is there a way to shorten the code here or make it more concise?

Reply:
best way I can do right now because this takes advantage of short circuiting

open to suggestions if you have any tho, i tried to shorten while doing before